docs/libs/ml/svm.md

```diff
@@ -27,34 +27,34 @@ under the License.
 
 ## Description
 
-Implements an SVM with soft-margin using the communication-efficient distributed dual coordinate 
-ascent algorithm with hinge-loss function. 
```

Comment:
I don't know what settings this it, but it would be helpful to turn off the space removal at the end of lines, because it makes it difficult to see the actual changes.